src/rollup-plugin-copy-components.ts

```diff
@@ -2,7 +2,8 @@ import fsExtra from 'fs-extra';
 import path from 'path';
 import { globSync } from 'glob';
 import { Plugin, PluginContext } from 'rollup';
-import { green } from 'colorette'
+import { green } from 'colorette';
+import chokidar from 'chokidar';
```

Comment:
Do you know if we need this dependency? How is it done in vue plugin where new files are detected??

Reply:
As discussed - got rid of it.